### PR TITLE
Feature/gaps richness

### PR DESCRIPTION
**Pull request by cazapatamar**

### CODE REVIEW

File: services/diversity/src/services/richness.js [hunk 1]
```diff
@@ -110,6 +110,48 @@ module.exports = () => {
           return data;
       }
     },
+
+    /**
+     * Get values for richness species gaps in the given area
+     *
+     * @param {String | Number} areaType area type
```
Comment (erikasv) [inline services/diversity/src/services/richness.js:117]: We keep making this mistake, areaType is always string, is areaId the one that can be number or string.

Also please fix the doc in the older two functions

File: services/diversity/src/services/richness.js [hunk 2]
```diff
@@ -110,6 +110,48 @@ module.exports = () => {
...
+     * @param {String} areaId area id
+     *
+     * @returns {Object} Values of richness species gaps
+     */
+    // eslint-disable-next-line no-unused-vars
```
Comment (erikasv) [inline services/diversity/src/services/richness.js:122]: don't disable eslint, just delete the parameters but leave them documented so we can add them when implement the real function

**Review by erikasv — CHANGES_REQUESTED** [review]

File: services/diversity/src/services/richness.js (hunk 1 above)
Comment (cazapatamar) [reply services/diversity/src/services/richness.js]: Fixed

File: services/diversity/src/services/richness.js (hunk 2 above)
Comment (cazapatamar) [reply services/diversity/src/services/richness.js]: Fixed

File: services/diversity/src/services/richness.js [hunk 3]
```diff
@@ -110,6 +110,48 @@ module.exports = () => {
...
+    /**
+     * Get values for richness species concentration in the given area
+     *
+     * @param {String | Number} areaType area type
+     * @param {String} areaId area id
+     *
+     * @returns {Object} Values of richness species concentration
+     */
+    // eslint-disable-next-line no-unused-vars
+    getConcentration: async (areaType, areaId) => {
```
[the remark this answers is not in the document]
Comment (cazapatamar) [reply services/diversity/src/services/richness.js]: Fixed

**Review by erikasv — APPROVED** [review]

**Pull request merged**